src/components/DigitalCredentials/CredentialsDashboard.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div>
+    <!-- Dialogs -->
+    <ConfirmRevokeCredentialDialog
+      :dialog="confirmRevokeCredentialDialog"
+      attach="#app"
+      @close="hideConfirmRevokeCredentialDialog"
+      @proceed="revokeCredential(issuedCredential)"
+    />
+
+    <ConfirmReplaceCredentialDialog
+      :dialog="confirmReplaceCredentialDialog"
+      attach="#app"
+      @close="hideConfirmReplaceCredentialDialog"
+      @proceed="replaceCredential(issuedCredential)"
+    />
+
+    <CredentialRevokedDialog
+      :dialog="credentialRevokedDialog"
+      attach="#app"
+      @close="credentialRevokedDialog = false"
+    />
+
+    <RevokeCredentialErrorDialog
+      :dialog="revokeCredentialErrorDialog"
+      attach="#app"
+      @close="revokeCredentialErrorDialog = false"
+    />
+
+    <v-container>
+      <v-row>
+        <v-col
+          cols="12"
+          md="8"
+        >
+          <CredentialsTable
+            v-if="issuedCredentials?.length"
+            :issuedCredentials="issuedCredentials"
+            @onPromptConfirmRevokeCredential="displayConfirmRevokeCredentialDialog"
+            @onPromptConfirmReplaceCredential="displayConfirmReplaceCredentialDialog"
+          />
+          <CredentialsLanding v-else />
+        </v-col>
+        <v-col
+          cols="12"
+          md="4"
+        >
+          <CredentialsInfo />
+        </v-col>
+      </v-row>
+    </v-container>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import { useBusinessStore } from '@/stores'
+import { Getter } from 'pinia-class'
+import { DigitalCredentialIF } from '@/interfaces'
+import { LegalServices } from '@/services'
+import CredentialsInfo from '@/components/DigitalCredentials/CredentialsInfo.vue'
+import CredentialsLanding from '@/components/DigitalCredentials/CredentialsLanding.vue'
+import CredentialsTable from '@/components/DigitalCredentials/CredentialsTable.vue'
+import ConfirmReplaceCredentialDialog from '@/components/DigitalCredentials/dialogs/ConfirmReplaceCredentialDialog.vue'
+import ConfirmRevokeCredentialDialog from '@/components/DigitalCredentials/dialogs/ConfirmRevokeCredentialDialog.vue'
+import CredentialRevokedDialog from '@/components/DigitalCredentials/dialogs/CredentialRevokedDialog.vue'
+import ReplaceCredentialErrorDialog from '@/components/DigitalCredentials/dialogs/ReplaceCredentialErrorDialog.vue'
+import RevokeCredentialErrorDialog from '@/components/DigitalCredentials/dialogs/RevokeCredentialErrorDialog.vue'
+import { Routes } from '@/enums'
+
+@Component({
+  components: {
+    CredentialsInfo,
+    CredentialsLanding,
+    CredentialsTable,
+    ConfirmRevokeCredentialDialog,
+    ConfirmReplaceCredentialDialog,
+    CredentialRevokedDialog,
+    ReplaceCredentialErrorDialog,
```

Comment:
This dialog does not appear to be used. Future, or delete the references?

Reply:
Added it to the UI

Reply:
Looks good.